components/pedigree/resources/src/main/resources/pedigree/view/workspace.js

```diff
@@ -139,17 +140,17 @@ define([
             // set display:block
             svgText = svgText.replace(/(<svg[^<>]+style=")/g, "$1display:block; ");
```

Comment:
@meatcar - those are all good ideas, but a bit more on performance: performance of 10 stacked regexps is not as good as say one regexp or something smarter, but slowdown is negligible (even if it is 10x, the base time is ~0). But (potentially) DOM manipulation for a large SVG is 100x or 1000x the time, and may become noticeable (e.g. just adding all possible years to a datepicker via DOM manipulation offscreen is noticeably slow, while composing HTML via javascript and doing one assignment is fast.). Virtual DOM should resolve this, but a straightforward implementation likely will become slow enough to worry about it, even if it is done once